pages/glossary/explicit-role.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Explicit Semantic Role
+key: explicit-role
+---
+
+The explicit semantic role is the [semantic role](#semantic-role) of an element set by its [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute).
+
+The [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute) takes a list of tokens. The explicit semantic role is the first valid role in this list. If none of the tokens are valid, the [implicit semantic role](#implicit-role) will be used instead.
+
+Non-abstract roles defined in the following specifications are considered valid:
+
+- [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+- [Digital Publishing WAI-ARIA Module 1.0](https://www.w3.org/TR/dpub-aria-1.0/)
+- [WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/)
+
+Other roles may be added as they become available. Not all roles will be supported in all assistive technologies. Testers are encouraged to adjust which roles are allowed according to the [accessibility support base line](https://www.w3.org/TR/WCAG-EM/#step1c). For the purposes of executing test cases in all rules, it should be assumed that all roles are supported by assistive technologies so that none of the roles fail due to lack of accessibility support.
+
+#### Acccessibility Support

```

Comment:
given the heading of each glossary terms are rendered as `h2` in the website, it would make sense that these are `h3` so we do not skip headings.

Reply:
That is a `####` in the current def of semantic role (which is why I copied it that way).
But yes, I agree that we shouldn't skip heading and use only `###`.

Reply:
Actually, the problem comes from the rule pages where "Referenced glossary" is a `h2`, and each glossary item is a `h3`, thus sub-headings like this one need to be `h4`.

It is annoying that glossary terms are rendered sometimes as `h2` and sometimes as `h3`…

Reply:
@jkodu do you have an opinion on the `h2`/`h3` inconsistence of glossary items I'm explaining in this discussion?

Reply:
For the sake of this PR, could we here do whatever is done (mostly) in other glossary terms, and then split out the task of creating consistency between the Glossary page and the embedded Glossary on the Rules pages?

Reply:
Lets get this merged here & will solve for heading inconsistencies in a different PR